src/gui/plugins/view_angle/ViewAngle.cc

```diff
@@ -73,11 +73,23 @@ namespace ignition::gazebo
     /// coming from qml side
     public: bool newCamClipDist = false;
 
+    /// \brief GUI camera horizontal FOV in degree
+    public: double camHFOV = 0;
```

Comment:
How about storing this as `math::Angle`? Then there's no ambiguity about the units

```suggestion
    public: math::Angle camHFOV = 0;
```

Reply:
Ah nevermind, this will make the property more complicated to use in the QML, so I think it's not worth the trouble.